### PR TITLE
README update

### PR DESCRIPTION
**Pull request by ziw-liu**
Update README before PyPI upload:

- Update reference
- Fix image links using full HTTP url instead of relative
- Update conda command since mamba is now unnecessary.

Test PyPI page: https://test.pypi.org/project/viscy/

### CODE REVIEW

**Comment by edyoshikun** [comment]
```
pip install -i https://test.pypi.org/simple/ viscy
Looking in indexes: https://test.pypi.org/simple/
Collecting viscy
  Downloading https://test-files.pythonhosted.org/packages/97/9e/b01d15ce37a0605396c7967f0ceff784510d2ec2db873facd4668287ad1a/viscy-0.1.0-py3-none-any.whl.metadata (12 kB)
INFO: pip is looking at multiple versions of viscy to determine which version is compatible with other requirements. This could take a while.
ERROR: Could not find a version that satisfies the requirement iohub==0.1.0 (from viscy) (from versions: 0.1.0.dev2, 0.1.0.dev4, 0.1.0.dev5)
ERROR: No matching distribution found for iohub==0.1.0
```

File: README.md [hunk 1]
```diff
@@ -138,19 +136,26 @@ The virtual staining method is described in [this preprint](https://doi.org/10.1
 }
 ```
 
-We also report the application of the virtual staining pipeline in [this preprint](https://doi.org/10.1101/2023.12.19.572435):
+The previous version of virtual staining was reported in [this paper](https://elifesciences.org/articles/55502):
```
Comment (edyoshikun) [inline README.md:139]: I think we can just list the citations and sort them with most recent first.

Comment (edyoshikun) [reply README.md:139]: Not strong opinion, but we will start having more CV applications in this repo

File: README.md [hunk 2]
```diff
@@ -8,20 +8,18 @@ Our pipeline also provides utilities to export the models to ONNX format for use
 We will grow the collection of the models suitable for high-throughput imaging and phenotyping.
 Expect rough edges until we release a PyPI package.
 
-![virtual_staining](docs/figures/phase_to_nuclei_membrane.svg)
+![virtual_staining](https://github.com/mehta-lab/VisCy/blob/main/docs/figures/phase_to_nuclei_membrane.svg?raw=true)
 
 This pipeline evolved from the [TensorFlow version of virtual staining pipeline](https://github.com/mehta-lab/microDL), which we reported in [this paper in 2020](https://elifesciences.org/articles/55502). The previous pipeline is now a public archive, and we will be focusing our efforts on VisCy.
```
Comment (edyoshikun) [inline README.md:13]: I would move this comment by the citation of the life paper or remove it. I don't think this links relevant.

File: README.md [hunk 3]
```diff
@@ -8,20 +8,18 @@ Our pipeline also provides utilities to export the models to ONNX format for use
 We will grow the collection of the models suitable for high-throughput imaging and phenotyping.
```
Comment (edyoshikun) [inline README.md:8]: This package contains a growing collection of models suitable for high-throughput imaging and phenotyping. 
Some applications include:
- robust virtual staining of cellular landmarks 
- infection classifier (WIP)

File: README.md [hunk 4]
```diff
@@ -8,20 +8,18 @@ Our pipeline also provides utilities to export the models to ONNX format for use
...
 Expect rough edges until we release a PyPI package.
```
Comment (edyoshikun) [inline README.md:9]: remove this line since we will be be adding this to PyPI.

File: README.md (hunk 1 above)
Comment (mattersoflight) [reply README.md:139]: I would a) shorten the reference and b) move it up to the section before `Installing VisCy`

Comment (mattersoflight) [reply README.md:145]: Implementing these changes.

File: README.md [hunk 5]
```diff
@@ -3,25 +3,29 @@
...
 The current focus of the pipeline is on the image translation models for virtual staining of multiple cellular compartments from label-free images.
-We are building these models for simultaneous segmentation of nuclei and membrane, which are the first steps in a single-cell phenotyping pipeline.
-Our pipeline also provides utilities to export the models to ONNX format for use at runtime.
-We will grow the collection of the models suitable for high-throughput imaging and phenotyping.
-Expect rough edges until we release a PyPI package.
+We are building these models for simultaneous segmentation of nuclei and membrane, which are the first steps in a single-cell phenotyping pipeline. 
 
-![virtual_staining](docs/figures/phase_to_nuclei_membrane.svg)
+This pipeline reports the models and training protocols reported in our recent [preprint on robust virtual staining](https://www.biorxiv.org/content/10.1101/2024.05.31.596901).
+
```
Comment (mattersoflight) [inline README.md:9]: A549 movie

File: README.md [hunk 6]
```diff
@@ -3,25 +3,29 @@
...
+This pipeline evolved from the [TensorFlow version of virtual staining pipeline](https://github.com/mehta-lab/microDL), which we reported in [this paper in 2020](https://elifesciences.org/articles/55502). The previous pipeline is now a public archive, and we will be focusing our efforts on VisCy. Our pipeline also provides utilities to export the models to ONNX format for use at runtime.
+
```
Comment (mattersoflight) [inline README.md:11]: Fly-through the HEK293T quantitative phase and zernike phase.

File: README.md [hunk 7]
```diff
@@ -3,25 +3,29 @@
...
+
```
Comment (mattersoflight) [inline README.md:12]: 4D movie of neuromast

**Comment by ziw-liu** [comment]
> @ziw-liu
> 
> ```
> pip install -i https://test.pypi.org/simple/ viscy
> Looking in indexes: https://test.pypi.org/simple/
> Collecting viscy
>   Downloading https://test-files.pythonhosted.org/packages/97/9e/b01d15ce37a0605396c7967f0ceff784510d2ec2db873facd4668287ad1a/viscy-0.1.0-py3-none-any.whl.metadata (12 kB)
> INFO: pip is looking at multiple versions of viscy to determine which version is compatible with other requirements. This could take a while.
> ERROR: Could not find a version that satisfies the requirement iohub==0.1.0 (from viscy) (from versions: 0.1.0.dev2, 0.1.0.dev4, 0.1.0.dev5)
> ERROR: No matching distribution found for iohub==0.1.0
> ```



> @ziw-liu
> 
> ```
> pip install -i https://test.pypi.org/simple/ viscy
> Looking in indexes: https://test.pypi.org/simple/
> Collecting viscy
>   Downloading https://test-files.pythonhosted.org/packages/97/9e/b01d15ce37a0605396c7967f0ceff784510d2ec2db873facd4668287ad1a/viscy-0.1.0-py3-none-any.whl.metadata (12 kB)
> INFO: pip is looking at multiple versions of viscy to determine which version is compatible with other requirements. This could take a while.
> ERROR: Could not find a version that satisfies the requirement iohub==0.1.0 (from viscy) (from versions: 0.1.0.dev2, 0.1.0.dev4, 0.1.0.dev5)
> ERROR: No matching distribution found for iohub==0.1.0
> ```

That's because iohub==0.1.0 was never uploaded to Test PyPI... Can you install from PyPI first?

**Comment by ziw-liu** [comment]
@mattersoflight adding all 3 movies will take a lot of space in README. I think links to a [wiki gallery](https://github.com/mehta-lab/VisCy/wiki/Gallery) are more suited there.

**Comment by edyoshikun** [comment]
Perhaps, it makes sense to play them either in a grid (merge the movies all in one canvas). I think a movie on the starting page is nice to have. Maybe then pick a favorite one if the prior is not possible.

File: README.md (hunk 3 above)
Comment (ziw-liu) [reply README.md]: Now moved to the top.

File: README.md (hunk 2 above)
Comment (ziw-liu) [reply README.md]: Removed.

**Comment by ziw-liu** [comment]
> Perhaps, it makes sense to play them either in a grid (merge the movies all in one canvas). I think a movie on the starting page is nice to have. Maybe then pick a favorite one if the prior is not possible.

Good idea! See https://github.com/mehta-lab/VisCy/pull/96/commits/c49cda6f7c9c22bd6e8a160409e9a000663d3387

**Review by edyoshikun — APPROVED** [review]
LGTM now! Thanks for the edits.

**Comment by mattersoflight** [comment]
lGtm (looks Great to me!). Merging.

**Pull request merged**